src/main.cpp

```diff
@@ -470,6 +464,16 @@ static bool setup_log_params(const Settings &cmd_args)
 	if (cmd_args.getFlag("verbose") || cmd_args.getFlag("trace"))
 		g_logger.addOutput(&stderr_output, LL_VERBOSE);
 
+	if (cmd_args.getFlag("trace")) {
+		dstream << _("Enabling trace level debug output") << std::endl;
+		g_logger.addOutput(&stderr_output, LL_TRACE);
+	}
+
+	if (cmd_args.getFlag("socketdebug")) {
+		dstream << _("Enabling socket debug output") << std::endl;
+		socket_enable_debug_output = true;
```

Comment:
Since this is a rare debug option I think leaving it together with `--trace` is okay